site/en/docs/webstore/discovery/index.md

```diff
@@ -51,5 +51,59 @@ the home page, items chosen for Editors' Picks must follow our general [best
 practices][best-practices] to be considered, but there is no set checklist of requirements to
 guarantee being featured. Developers cannot pay to be featured as an Editors' Pick.
 
-[best-practices]: https://developer.chrome.com/docs/webstore/best_practices/
-[best-listing]: https://developer.chrome.com/docs/webstore/best_listing/
+## Item badges {: #badges }
+
+Badges on Chrome Web Store are visible to users as they browse the store, and offer an added signal
+of quality and trust on vetted items. Developers who earn these badges may receive higher rankings
+in search and filtering, and might also see their extensions appear in special promotions both on
+and off Chrome Web Store. Learn more about the badges below.
+
+### Featured badge {: #featured-badge }
+
+{% Img src="image/WlD8wC6g8khYWPJUsQceQkhXSlv1/FL4P4tqG0jrhTdLuQ4Jp.png", width="800", height="400",
+   alt="Screenshot of an extension listing that has a featured badge.", class="screenshot" %}
+
+The Featured badge is assigned to extensions that follow our technical best practices and meet a
+high standard of user experience and design. Chrome team members manually evaluate each extension
+before it receives the badge, paying special attention to the following:
+
+1. Adherence to Chrome Web Store’s [best practices][best-practices] guidelines, including providing
+   an enjoyable and intuitive experience, using the latest platform APIs and respecting the [privacy
+   of end-users][user-privacy].
+2. A [store listing page][best-listing] that is clear and helpful for users, with quality images and
+   a detailed description.
+
+### Established Publisher badge {: #publisher-badge }
+
+{% Img src="image/WlD8wC6g8khYWPJUsQceQkhXSlv1/vZQXBcGDZaWBWQLPCM95.png", width="800", height="400",
+   alt="Screenshot of an extension listing that has an established publisher badge.",
+   class="screenshot" %}
+
+The Established Publisher badge showcases developers who have verified their identity and demonstrated compliance with the developer program policies. This badge is granted automatically to publishers who meet the following two conditions:
+
+1. The publisher's identity has been [verified][verified].
+2. The publisher has established a consistent positive track record with Google services and
+   compliance with the Developer Program Policy.
+
+Any extensions built by a developer who follows the Chrome Web Store Developer Program Policies and
+does not have any unresolved violations will be considered. For new developers, it will take at
+least a few months of respecting these conditions to qualify. Today, this represents nearly 75% of
```

Comment:
>at least a few months

Is there no way to be more specific?

Reply:
Good question. So far, no, but I will follow up on this.